Fixes the following error:
chan_phone.c:41:29: error: linux/compiler.h: No such file or directory
--- asterisk-1.2.13/channels/chan_phone.c.orig	2006-10-21 14:14:02.000000000 +0200
+++ asterisk-1.2.13/channels/chan_phone.c	2006-10-21 14:14:17.000000000 +0200
@@ -37,9 +37,6 @@
 #include <linux/telephony.h>
 /* Still use some IXJ specific stuff */
 #include <linux/version.h>
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(2,6,0)
-# include <linux/compiler.h>
-#endif
 #include <linux/ixjuser.h>
 
 #include "asterisk.h"
